tests/datasets/test_dali.py

```diff
@@ -4,11 +4,10 @@
 try:
     import DALI
 except ImportError:
-    logging.error(
+    raise ImportError(
```

Comment:
Yeah, the same thing as above:
"An error occured when importing this dataset. Most likely this is due to "